### ISSUE TITLE
CanonicalBibtexEntryTest fails locally

### ISSUE THREAD

**Issue by stefan-kolb** [JabRef/jabref#493]
For me the test fails locally but runs on the CI server

Expected
```  
@article{key,[]
  author = {abc},
 ...
```

Actual

```  
@article{key,[
]
  author = {abc},
 ...
```

Newline problem inside canonical formatter?

**Comment by tobiasdiez** [comment]
I have this too, with a dozen of other tests failing too. As far as I can see, they are all caused by my customized preferences.

**Comment by simonharrer** [comment]
Hm, canonical should be independent from preferences...

**Comment by simonharrer** [comment]
Should be fixed in e98c7db01df954ddd6082ca46bb9ef275d546baa
